presto-docs/src/main/sphinx/connector/hive.rst

```diff
@@ -310,6 +310,13 @@ Property Name                                      Description
                                                    can be used to use a different location for each user.
 
 ``hive.translate-hive-views``                      Enable translation for Hive views. (experimental)            ``false``
+
+``hive.legacy-hive-view-translation``              Use the legacy algorithm to translate Hive views.            ``false``
```

Comment:
I think the descriptions of `hive.translate-hive-views` and `hive.legacy-hive-view-translation` might be confusing for new users, since we mention that the first property is as experimental itself and mention the second one as "legacy". Also, I think we say experimental, because there're chances that things may evolve/break.

@electrum Can we take an approach where we "failover" to the legacy behavior when coral fails? This can be placed at https://github.com/prestosql/presto/blob/master/presto-hive/src/main/java/io/prestosql/plugin/hive/ViewReaderUtil.java#L183 .

Reply:
cc @sopel39

Reply:
It might be even better to have one toggle only .. 

```
hive.translate-hive-views=false // off
hive.translate-hive-views=full // or some other value for coral
hive.translate-hive-views=legacy // for old one
hive.translate-hive-views=true // would have to sort out what that defaults to .. full or legacy (legacy would be backward compatible.. but less ideal going forward
```

Reply:
> @electrum Can we take an approach where we fallback to the legacy behavior when coral fails?

Even if we have this I would rather have a way to force old logic. There still can be a case when Coral logic seems to work (automatic fallback does not trigger), but generated translation is not correct.

Reply:
Also, unless we want to change implementation before the `348` release is cut, we should document current state. And this PR is about that.

Reply:
Correct. Let's make sure the documentation is merged before the release.